indexing.rdf

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
+<rdf:RDF 
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+
+    <owl:Ontology rdf:about="http://fedora.info/definitions/v4/indexing#">
+        <rdfs:label xml:lang="en">Fedora Commons Indexing Ontology</rdfs:label>
+        <rdfs:comment xml:lang="en">Ontology for user-defined indexing properties and classes on Fedora-curated resources.</rdfs:comment>
+    </owl:Ontology>
+
+    <owl:DatatypeProperty rdf:about="http://fedora.info/definitions/v4/indexing#hasIndexingTransformation">
```

Comment:
You can define entities in XML, and use those to shorten the URIs.

``` xml
<!DOCTYPE rdf:RDF [
    <!ENTITY indexing "http://fedora.info/definitions/v4/indexing#" >
]>
...
<owl:DatatypeProperty rdf:about="&indexing;hasIndexingTransformation">
```

But I think it's fine to leave the full URIs.  I don't think the shortening makes it much easier to read.  Also, I'd rather stick with RDF/XML, or at least provide RDF/XML as one of a couple formats, since it seems to be the most common format for ontology docs.